drone_node/include/utils/geodetic_converter.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef RVIZ_AERIAL_PLUGINS_GEODETIC_CONVERTER_HPP_
```

Comment:
This include guard is wrong. Did the code get copied over? Should we be refactoring this into somewhere more convenient?

Reply:
The code is originally from this repo. I included in the header https://github.com/osrf/drone_demo/blob/1e7837279fd0af83201f259564b22f91d45f4a39/drone_node/include/utils/geodetic_converter.hpp

How should I include this?